ui/packages/catalog-ui-search/src/main/webapp/js/model/Query.js

```diff
@@ -316,6 +316,9 @@ Query.Model = PartialAssociatedModel.extend({
       })
     })
   },
+  async cqlSanitizePlugin(data) {
```

Comment:
remove `async`

Reply:
also remove `data` from the method signature, and just `return` (do nothing) in this method. don't need to return `data`